openlibrary/templates/type/author/edit.html

```diff
@@ -21,9 +21,9 @@ <h1>$_("Edit Author")</h1>
 
     <form id="addAuthor" name="edit" method="post" action="" class="olform books validate">
 
-
-    <div class="formElement title">
-        <div class="formElement">
+    <!-- TODO: Where should these styles go (assuming we want to display like this) -->
+    <div class="formElement title" style="display:flex">
+        <div class="formElement" style="flex-basis: calc(2 / 3 * 100%);">
```

Comment:
Not the best place for it, but maybe toss it in olform under `.olform#addAuthor` for now.